CHANGE_LOG

```diff
@@ -1,3 +1,19 @@
+v0.27.0
+-------
+
+This release updates llvmlite to LLVM 7.  Note that LLVM 7.0.0 contains
+a critical bug that is resolved with a patch included in the `llvmdev conda
+package recipe`_.  The final release of LLVM 7.0.1 may also resolve the issue.
+
+* PR #434: Add another thread for RPi builds. 
+* PR #430: llvm lld integration, merge #428
+* PR #428: Build LLD as part of the llvmdev package
```

Comment:
This got merged as part of #430, but that itself was a merge so I'm electing to leave it declared as is.